src/Umbraco.Core/Services/AuditService.cs

```diff
@@ -28,37 +28,15 @@ public AuditService(
         ILoggerFactory loggerFactory,
         IEventMessagesFactory eventMessagesFactory,
         IAuditRepository auditRepository,
-        IUserService userService,
+        IUserIdKeyResolver userIdKeyResolver,
```

Comment:
Be aware that this is a breaking change, but without it we would run into circular dependency exceptions.